README.md

```diff
@@ -6,14 +6,15 @@
 
 ## Windows
 
-Once you've cloned the repo locally on your machine, open a command line/power shell in that folder and run ```yarn``` or ```npm install``` to install the packages for the app.
+Make sure you download and install NodeJS with the version between 12.X and 14.X, we recommend 14.17.2. The latest versions are not supported.
+
+Once you've cloned the repo locally on your machine, open a command line/power shell in that folder and run ```yarn``` to install the packages for the app.
```

Comment:
when putting commands in docs you can use:
```
```bash
command
` ` ` <- without spaces :))
```

this way github will display a copy button for each comamnd, and the doc is more clean